docs/HACKING.md

```diff
@@ -33,7 +37,7 @@ The commands given below are for use by core contributors. If you are a communit
     $ cd ~/devel
     $ git clone https://github.com/Solinea/goldstone-server.git
```

Comment:
The preceding sentence refers to, "clone commands," but there's only one clone command.